src/scene/shader-lib/chunks/standard/frag/frontend.js

```diff
@@ -0,0 +1,37 @@
+export default /* glsl */`
+struct Frontend {
+    vec2 dUvOffset;
```

Comment:
do we need to keep 'd' prefix now that these are not global?

Reply:
these should be easy to understand names as users will providing values to these at some point in custom shader front ends .. so lets see if we can have a nice consistent clear naming.

Reply:
We could dump the d prefix and change the clearcoat and sheen to be prefixed differently, like clearcoat_Specularity and sheen_Glossiness.

Reply:
and perhaps no underscores. just `clearcoatSpecularity`.
even ao could maybe expanded to ambentOcclusion, or maybe just occlusion?
and normalW to worldNormal, or normalWorld.

perhaps make a proposal here before changing all the chunks.